### DISCUSSION TITLE
PCB Snes

### DISCUSSION THREAD

**Discussion by romehi** [sanni/cartreader#271] [Need help]
Bom dia!

em vários tópicos de discussões vi que o pessoal esta usando essa PCB de baixo, gostaria de saber se alguém tem o arquivo dela para que consiga faze-la.

Pois estou com dificuldades em achar PCB
<img width="467" alt="pcb" src="https://user-images.githubusercontent.com/84623440/136392540-dfbe2a62-10ff-47cf-b88d-35d4a20ee42b.png">

**Comment by sanni** [comment]
It's a PCB from a Chinese SNES game from Aliexpress. No files available.

**Comment by StanMOI** [comment]
Yes ,
PCB’s are from China, Aliexpress sellers .
Different flash size, different fram size.
Ask sellers to get picture of PCB before to buy it.

Be careful, those PCB don’t have chips to do «translation» of signals (5v<->3v).
So it’s can slowly damage your SNES  console and Pcb.
It’s out of SNES specifications cause they use 3v chip in 5v system with  just one chip for voltage conversion .
